pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -593,6 +605,7 @@ mod minting {
 						avatar_ids: vec![AwesomeAvatars::owners(ALICE)[0]],
 					},
 				));
+				assert_eq!(Balances::total_balance(&ALICE), 1_234_017_890_123_456u64);
```

Comment:
After adding `ExtBuilder::mint_fees`:
```suggestion
				initial_balance -= mint_fees.one;
				assert_eq!(Balances::total_balance(&ALICE), initial_balance);
```